### PR TITLE
Add bump-brew-formula workflow

### PR DESCRIPTION
**Pull request by ronakfof**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/bump-brew-formula.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        run: echo "RELEASE_TAG=${GITHUB_REF/refs\/tags\/}" >> $GITHUB_ENV
+      - name: Configure git user name and email
+        uses: Homebrew/actions/git-user-config@master
+        with:
+          username: aws-viewer-for-cbmc-release-ci
+      - name: Create homebrew PR
+        run: |
+          brew update-reset
+          brew tap aws/tap
+          brew bump-formula-pr --tag "$RELEASE_TAG" --revision "$GITHUB_SHA" aws/tap/litani
```
Comment (markrtuttle) [inline .github/workflows/bump-brew-formula.yaml:21]: Is this a typo?  litani -> cbmc-viewer

File: .github/workflows/tagged_release_message.md [hunk 2]
```diff
@@ -1,10 +1,16 @@
 This is CBMC Viewer version {}.
 
-To install this release, download the "PIP installation package" below and run
+To install this release using brew, run
+```
+  brew install aws/tap/cbmc-viewer
+```
+
+To install this release using pip, download the "PIP installation package" below and run
```
Comment (markrtuttle) [inline .github/workflows/tagged_release_message.md:8]: I prefer to say

On MacOS, you can install with brew:  Run
```
brew install aws/tap/cbmc-viewer
```
On all machines, you can install with pip: Download the "PIP installation package" below and run

**Review by markrtuttle — CHANGES_REQUESTED** [review]

File: .github/workflows/bump-brew-formula.yaml (hunk 1 above)
Comment (ronakfof) [reply .github/workflows/bump-brew-formula.yaml]: Resolved

File: .github/workflows/tagged_release_message.md (hunk 2 above)
Comment (ronakfof) [reply .github/workflows/tagged_release_message.md]: Resolved

File: .github/workflows/tagged_release_message.md [hunk 3]
```diff
@@ -1,10 +1,16 @@
...
+```
+  brew install aws/tap/cbmc-viewer
+```
+
+On all machines, you can install with pip: Download the "PIP installation package" below and run
 ```
   sudo python3 -m pip install --upgrade cbmc_viewer-*.whl
 ```
 
+In addtion to the PIP package -
```
Comment (markrtuttle) [inline .github/workflows/tagged_release_message.md:13]: Delete this line?  Is this a mistake?

File: .github/workflows/bump-brew-formula.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+on:
+  release:
+    types: [created]
```
Comment (markrtuttle) [inline .github/workflows/bump-brew-formula.yaml:3]: release.yaml for creating the release page is triggered by 

```
# A new release is triggered by a new tag of the form viewer-VERSION
on:
  push:
    tags:
      - viewer-*
```

Does this need to be the same?  Setting a tag creates the release page.  Does that trigger a release event and trigger this workflow?

**Review by markrtuttle — CHANGES_REQUESTED** [review]

File: .github/workflows/bump-brew-formula.yaml (hunk 4 above)
Comment (ronakfof) [reply .github/workflows/bump-brew-formula.yaml:3]: Yes it'll trigger this workflow once release is created.
Examples - https://github.com/awslabs/aws-build-accumulator/tree/release/.github/workflows

**Review by markrtuttle — APPROVED** [review]

File: .github/workflows/bump-brew-formula.yaml (hunk 4 above)
Comment (ronakfof) [reply .github/workflows/bump-brew-formula.yaml:3]: One more thing is to be updated. Release should be created using the bot account and not  GitHub actions.  Will update.

**Pull request merged**